src/App.js

```diff
@@ -1,25 +1,50 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import "./Styles/sass/styles.scss";
+// Sections Components
+import Header from "./Components/Header";
+import DisplayEvents from "./Components/DisplayEvents";
+import Footer from "./Components/Footer";
+import UserForm from "./Components/UserForm";
 
 function App() {
```

Comment:
since your App.js looks a bit full, perhaps you could create a separate component (i.e. landing page) to store some of the components that mount on load, and keep your App.js more empty